cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -331,6 +331,12 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
```

Comment:
i'm pretty sure these should be under `rules:`.
http://people.redhat.com/jrivera/openshift-docs_preview/openshift-online/glusterfs-review/rest_api/apis-rbac.authorization.k8s.io/v1.ClusterRole.html#object-schema

i guess this applies to the rest of the files that are changed from the diff.

Reply:
@neolit123, they are under `rules:` as one more list item. Am i missing something here?

Reply:
never mind, i got confused by the indentation.